ddtrace/tracer/spancontext.go

```diff
@@ -25,7 +25,7 @@ type spanContext struct {
 
 	trace  *trace // reference to the trace that this span belongs too
 	span   *span  // reference to the span that hosts this context
-	drop   bool   // when true, the span will not be sent to the agent
+	drop   bool   // when true, the span will not be sent to the agent, even when not computing stats in the tracer.
```

Comment:
```suggestion
	drop   bool   // reports whether this span was dropped by the rate sampler
```
Let's be crystal clear, now that we're adding more flags which can confuse things.